### DISCUSSION TITLE
Trident 9440 not fully emulated?

### DISCUSSION THREAD

**Discussion by Ashir555** [86Box/86Box#2694] [Q&A]
Hi, I don't know if I can post this here but I was wondering if anyone knows why on PCEm when you use a Triden 9440 as video card before bios screen, the Trident shows a screen while in 86Box this is absent?

Can this be enabled somehow or is the Trident 9440 not fully emulated?

**Comment by TC1995** [comment]
the Trident 9440 is emulated as best as possible, also we are using a different bios version of said chip.

**Reply by Ashir555** [reply]
Oh, I see, so it's a video card bios thing...

Well, thank you answering.

Will that screen be implemented someday?, do you know?

**Reply by TC1995** [reply]
it's not needed at all.

**Reply by Ashir555** [reply]
Well yes, it's not needed but it's more accurate.

When I first try PCEm it was a real blast from the past see that pre bios screen. That was my very first video card. So many memories...

**Reply by OBattler** [reply]
It's, if anything, less accurate. On real hardware, it depends on when the specific machine BIOS goes to initialize PCI devices. Not all BIOS'es do it before the POST screen. PCem starts the card with a pre-initialized video BIOS mapping, but that's incorrect for PCI.

**Reply by Ashir555** [reply]
Well I use that card with 3 different P1 motherboards and it always show up that pre bios screen. I don't think it's a PCEm approach, it just like it worked back then. But then again, if my memory doesn't fail me, the Trident 3D Blade show a similar screen with the penultimate p1 motherboard I had, but with the last one that screen never show up, so I guess you are right.

By what you are saying, maybe there is some option I can enable on BIOS to have the same effect? (sorry, english is not my native languaje)

OBattler awesome work with 86Box btw. I really enjoy the variety of motherboards available. I even find a motherboard that uses the same AMI BIOS my very first P1 had, with the windowed interface. Back in 97 my neightborhood friends were like "woow look at that!". Thank you for the remembering!
